### ISSUE TITLE
Class variable as type alias

### ISSUE THREAD

**Issue by timrid** [microsoft/pylance-release#1043]
## Environment data

-   Language Server version: Pylance v2021.3.1
-   OS and version: Windows 10
-   Python version: Python 3.8.6

## Actual behaviour

The following code generates an error for `Color2`, because it is not recognised as type alias. But `Color1` is correctly recognised as type alias, and generates no error.

## Code Snippet / Additional information

```python
from typing import Literal

# ----------------------------------
Color1 = Literal[1, 2]
class Window1():
    def get_color(self) -> Color1: ...

c1 = Window1().get_color()
reveal_type(c1)

# ----------------------------------
class Window2():
    Color2 = Literal[3, 4]
    def get_color(self) -> Color2: ...

c2 = Window2().get_color()
reveal_type(c2)
```

![image](https://user-images.githubusercontent.com/6593626/111027736-7731aa80-83f2-11eb-87aa-72c512a6b732.png)

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release of pylance.

Type aliases in Python are tricky because they use the same syntax as variable assignments. A type checker needs to apply various heuristics to to tell them apart. ([PEP 613](https://www.python.org/dev/peps/pep-0613/) introduces a way to explicitly annotate type aliases, but implicit type aliases must still be supported.) This issue was the result of one of those heuristics that needed to be adjusted to accommodate type aliases defined within a class with literal types.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202132-17-march-2021
